easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-foss-2016b.eb

```diff
@@ -0,0 +1,44 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1706'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/openfoamplus/files/%(version)s',
+     'download')
+]
```

Comment:
style nitpicking: let's just make this a single line? I don't see the need to spread this over 4 lines...

```python
source_urls = [('https://sourceforge.net/projects/openfoamplus/files/%(version)s', 'download')]
```